next.config.ts

```
@@ -1,7 +1,52 @@
 import type { NextConfig } from "next";
 
+import type { NextConfig } from "next";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'NextConfig'. Consider to delete it or rename it.

'NextConfig' is defined here:


(lint/suspicious/noRedeclare)